src/praveen/narcissistic.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Function to find that the given number is Narcissistic Number or not
+ * @param {number} num number to check if it is Narcissistic Number
+ * @return {boolean} true or false
+ */
+function checkIfNarcissisticNumber(num: number): boolean {
+  if (num <= 0) {
+    return false;
+  }
```

Comment:
`0` is actually a narcissistic number. Condition shall not include `0`

Reply:
Thank you Raj, I have updated the code and unit test for the same.